recipe/meta.yaml

```
@@ -16,16 +16,18 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake
     - make
-  run_constrained:
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+  host:
+    - zlib
```

```suggestion
    - zlib
    - bzip2
```

The build environment is getting the wrong symbols, so add `bzip2` to the host environment (where all the shared objects needed for a build should be).

```
/home/conda/feedstock_root/build_artifacts/hepmc3_1724281635578/_build_env/bin/../lib/gcc/aarch64-conda-linux-gnu/13.3.0/../../../../aarch64-conda-linux-gnu/bin/ld: /home/conda/feedstock_root/build_artifacts/hepmc3_1724281635578/_build_env/lib/libbz2.so: error adding symbols: file in wrong format
collect2: error: ld returned 1 exit status
```